tests/inference_test.py

```diff
@@ -156,11 +156,16 @@ def rescale(img, input_height, input_width):
     init_net.RunAllOnGPU(use_gpu_engine=True)
     predict_net.RunAllOnGPU(use_gpu_engine=True)
 
+miopen_op_map = {'MaxPool':'MaxPool2D','AveragePool':'AveragePool2D'}
```

Comment:
We don't need this hack once PR#125 is merged

Reply:
removed the hack